src/Mado/QueryBundle/Queries/QueryBuilderFactory.php

```diff
@@ -474,10 +476,10 @@ public function sort()
         return $this;
     }
 
-    public function getQueryBuilder()
+    public function getQueryBuilder() :QueryBuilder
     {
         if (!$this->qBuilder) {
-            throw new \RuntimeException(
+            throw new UnInitializedQueryBuilderException(
                 "Oops! Query builder was never initialized! call ::createQueryBuilder('entityName', 'alias') to start."
```

Comment:
This message should be placed as protected property of "UnInitializedQueryBuilderException" class.